src/app.js

```diff
@@ -1,9 +1,15 @@
+require('dotenv').config()
 var express = require('express');
+var http = require('http')
 
-const app = express();
-
-app.get('/', (req, res) => res.send('Hello World!'))
+var models = require('./models/index')
 
-app.listen({ port: 8000 }, () => {
+const app = express();
+const httpServer = http.createServer(app);
```

Comment:
I plan to use websocket to add simple functions to make the user experience better. Like increasing or decreasing the item quantity as other user add or remove items from the cart, so if an item was sold out and someone removed it from his cart it would automatically increase the quantity for the user in that item page without the need to reload the it.

Reply:
A nice touch!